dist/api_data.py

```diff
@@ -1342,6 +1342,9 @@ def __ge__(self, other):
 'WT_SESSION.close' : Method([]),
 
 'WT_SESSION.compact' : Method([
+    Config('free_space_target', '20MB', r'''
+        minimum amount of space recoverable for compaction to proceed''',
+        min='0'),
```

Comment:
How about restricting the minimum value to 1MB?

Reply:
I'd be also inclined to keep the minimum as 1 MB. It doesn't make sense to compact files with free space that's less than that.

Reply:
Restricted the min value to 1MB here: 3b7c232369863d755900f3d7b59c0c5b18f8d7ef